subprojects/core/src/integTest/groovy/org/gradle/normalization/ConfigureRuntimeClasspathNormalizationIntegrationTest.groovy

```diff
@@ -339,6 +339,44 @@ class ConfigureRuntimeClasspathNormalizationIntegrationTest extends AbstractInte
         executedAndNotSkipped(project.customTask)
     }
 
+    @ToBeFixedForConfigurationCache(because = "classpath normalization")
+    def "properties files are normalized against changes to whitespace and comments"() {
```

Comment:
...and order, no?

Reply:
I don't see the whitespace change, what am I missing?

Reply:
I had intended to have a change to all three (whitespace, order and comments) but I think getting the ordering right sent me down a rabbit hole and then I forgot to finish up adding whitespace to the test.  Fixed.